files/lpasspw

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo -n "$*: " >/dev/null
+answer=`cat $LPASS_HOME/pw`
```

Comment:
Can this script check for the existence of such a password file and resume normal behavior of prompting the user or exploding if the file is not there?

I'm just thinking of users other than root trying to interactively use the lastpass cli and not having a password file. I don't want to break their experience.